### PR TITLE
Added prototype frontend to the chat.

### PR DESCRIPTION
**Pull request by TrevorFlanigan**
# Changes
Displays messages on the window instead of the console.
No usernames are currently being displayed, instead, everyone's name is displayed as a placeholder.
# Tickets
#51 #74

### CODE REVIEW

File: frontend/src/components/Chat.tsx [hunk 1]
```diff
@@ -10,75 +16,101 @@ const styles = {
   }
 };
 
+interface Message {
```
Comment (cowmanjoe) [inline frontend/src/components/Chat.tsx]: Can we move this interface to its own file in `frontend/src/models`?

File: frontend/src/components/Chat.tsx [hunk 2]
```diff
@@ -10,75 +16,101 @@ const styles = {
...
-          placeholder="What is your name?"
-          onChange={this.onChange}
-          onKeyUp={this.enterPressed}
-          value={this.state.message}
-        />
-      </div>
-    );
+    const { classes } = this.props;
+
     const chatWindow = (
```
Comment (cowmanjoe) [inline frontend/src/components/Chat.tsx]: `chatWindow` makes me think this is where the chat messages are displayed, not the field where you enter your chat text. Maybe `chatField` would be a better name

File: frontend/src/components/Chat.tsx [hunk 3]
```diff
@@ -10,75 +16,101 @@ const styles = {
...
           value={this.state.message}
         />
       </div>
     );
     return (
-      <div style={styles.chatBox}>
-        <h3>Chat:</h3>
-        {this.state.signedIn ? chatWindow : signInWindow}
+      <div style={{ paddingLeft: "2vw" }}>
+        {chatWindow}
```
Comment (cowmanjoe) [inline frontend/src/components/Chat.tsx]: I think it feels more natural to put the chat field below where the messages are displayed

File: frontend/src/components/Chat.tsx [hunk 4]
```diff
@@ -10,75 +16,101 @@ const styles = {
...
-        />
-      </div>
-    );
+    const { classes } = this.props;
+
     const chatWindow = (
       <div>
-        <input
-          placeholder="Send a message"
+        <TextField
```
Comment (cowmanjoe) [inline frontend/src/components/Chat.tsx]: Can we put a border around this field just to make it a bit more visible

**Comment by srijonsaha** [comment]
Now that @nuxeric PR's in, you can start using the real usernames for chat.

**Comment by TrevorFlanigan** [comment]
username integration is done

**Pull request merged**